Description: Add the non-multiarch include dir to pkg-config cflags for
 compatibility.
 Some projects only use sdl2-config/sdl2 pkg-config and expect to find the
 headers of other SDL2 libraries (libsdl2-image etc) in the same directory.
Author: Felix Geyer <fgeyer@debian.org>

diff --git a/sdl2.pc.in b/sdl2.pc.in
--- a/sdl2.pc.in
+++ b/sdl2.pc.in
@@ -12,4 +12,4 @@ Requires:
 Conflicts:
 Libs: -L${libdir} @SDL_RLD_FLAGS@ @SDL_LIBS@
 Libs.private: @SDL_STATIC_LIBS@
-Cflags: -I${includedir}/SDL2 @SDL_CFLAGS@
+Cflags: -I${includedir}/SDL2 -I${prefix}/include/SDL2 @SDL_CFLAGS@
